glossary.html

```diff
@@ -7,34 +7,16 @@
     <meta name="author" content="liamleung" />
     <meta name="description" content="CSD120 training website" />
     <title>CSD120-LiamLeung</title>
-    <link
-      rel="apple-touch-icon"
-      sizes="180x180"
-      href="Desktop/CSD120/apple-touch-icon.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="Desktop/CSD120/favicon-32x32.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="16x16"
-      href="Desktop/CSD120/favicon-16x16.png"
-    />
-    <link rel="manifest" href="Desktop/CSD120/site.webmanifest" />
-    <link
-      rel="mask-icon"
-      href="Desktop/CSD120/safari-pinned-tab.svg"
-      color="#5bbad5"
-    />
-    <link rel="shortcut icon" href="Desktop/CSD120/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="favicon/favicon_package_v0.16/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon_package_v0.16/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon_package_v0.16/favicon-16x16.png">
+    <link rel="manifest" href="favicon/favicon_package_v0.16/site.webmanifest">
+    <link rel="mask-icon" href="favicon/favicon_package_v0.16/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="favicon/favicon_package_v0.16/favicon.ico">
```

Comment:
The instructions were to place your favicon package contents at the root of your site.  You placed yours in a folder.  This is fine, but you then need to update your URLs to match